package.json

```diff
@@ -15,39 +15,40 @@
   "author": "",
   "license": "ISC",
   "engines": {
-    "node": ">=12.1.0 <19",
+    "node": ">=12.22.0 <19",
```

Comment:
Out of curiosity, is there any reason we cap Node versions at 18.9? I see the `>=12.22.0` is from Next.js v12, but is there anything in Node 19 that would break this app?

Reply:
node v19 hasn't been released yet so it is untested. Once v19 is released, we can test and update the engines